src/app/layout.tsx

```diff
@@ -1,6 +1,8 @@
 import type { Metadata } from "next";
```

Comment:
For Assignment i think it should be fine but as much as possible we will have compressed image in KB size